crates/ursa-network/src/behaviour.rs

```diff
@@ -130,12 +135,7 @@ pub enum BehaviourEvent {
 ///
 /// The events generated [`BehaviourEvent`].
 #[derive(NetworkBehaviour)]
-#[behaviour(
```

Comment:
"out_event" and "poll_method" have been removed in the new release so I had to refactor this.